apps/web/src/app/(dashboard)/documents/upload-document.tsx

```diff
@@ -81,7 +79,7 @@ export const UploadDocument = ({ className }: UploadDocumentProps) => {
     <div className={cn('relative', className)}>
       <DocumentDropzone
         className="min-h-[40vh]"
-        disabled={remaining.documents === 0 || !session?.user.emailVerified}
+        disabled={remaining.documents === 0}
```

Comment:
```suggestion
        disabled={remaining.documents === 0 || !session?.user.emailVerified}
```

@adithyaakrishna Revert this before finishing

Reply:
Reverted this in badbce44